files/config/default/etc/org.opencastproject.serviceregistry.impl.JobDispatcher.cfg

```diff
@@ -5,4 +5,4 @@
 # node is 2. Service registry dispatching is automatically set to 0 on everything but admin or allinone nodes and should
 # usually not be activated on these nodes to avoid concurrency problems.
 # Default: 0
-dispatch.interval=0
+dispatch.interval=2
```

Comment:
Service registry dispatching must not be enabled on anything except the admin node AFAIK, otherwise multiple servers try to dispatch jobs at the same time causing lock wait timeouts and conflicts.